### PR TITLE
build: update style lint

### PR DESCRIPTION
**Pull request by coryrylan**
## PR Checklist

Please check if your PR fulfills the following requirements:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

* [ ] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [x] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] clarity.design website / infrastructure changes
* [ ] Other... Please describe:

## What is the current behavior?
The style lint currently only understands CSS and errors a lot of Sass syntax incorrectly. This makes the build output noisy with warning log statements.

Issue Number: N/A

## What is the new behavior?
The build logs are now cleaned up and don't have all the sass warnings. The style linter now errors instead of warns and understands Sass syntax. Two of the default errors are disabled as they cause a lot of noise and will require more refactoring of component styles at a latter time.
Deletions in styles for this PR are primarily duplicate properties or imports.

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "test:format": "prettier --list-different './**/*.{js,json,md,ts,scss}'",
     "pretest:lint": "tsc -p tests/tslint/tsconfig.json",
     "test:lint:ts": "tslint -p . -c tslint.json \"src/**/*.ts\" -e \"src/schematics/**\"",
-    "test:lint:styles": "stylelint \"src/**/*.scss\" \"src/**/*.css\"",
```
Comment (gnomeontherun) [inline package.json]: Doesn't this now just ignore CSS files?

File: src/clr-angular/utils/_variables.clarity.scss [hunk 2]
```diff
@@ -137,6 +137,3 @@
 
 // Accordion
 @import '../accordion/variables.accordion';
-
-// Typography
-@import '../typography/variables.typography';
```
Comment (gnomeontherun) [inline src/clr-angular/utils/_variables.clarity.scss]: What is going on here?

File: src/clr-angular/utils/_components.clarity.scss [hunk 3]
```diff
@@ -44,9 +44,6 @@
 @import '../typography/typography.clarity';
 @import '../typography/code.clarity';
 
-// A11Y
-@import '../utils/a11y.clarity';
```
Comment (gnomeontherun) [inline src/clr-angular/utils/_components.clarity.scss]: Don't we need this?

File: package.json (hunk 1 above)
Comment (coryrylan) [reply package.json]: That was a typo from testing, there is one CSS file left in the project the 404 page which I didnt convert to Sass. I could convert it and then there would be no CSS files left or just add back the lint.

File: src/clr-angular/utils/_variables.clarity.scss (hunk 2 above)
Comment (coryrylan) [reply src/clr-angular/utils/_variables.clarity.scss]: Duplicate import, was imported already at the top of the file.

File: src/clr-angular/utils/_components.clarity.scss (hunk 3 above)
Comment (coryrylan) [reply src/clr-angular/utils/_components.clarity.scss]: Duplicate import, was imported already at the top of the file.

**Comment by coryrylan** [comment]
@Jinnie The scss deletions are duplicate properties or imports that already exist. The block deleted from the home page is from duplicate unnecessary statements. Duplicate statements (background) for things like gradients are already automatically done by the Angular CLI via a post css plugin. We technically don't event need that since our browser support does not even go back that far.

File: package.json (hunk 1 above)
Comment (coryrylan) [reply package.json]: I went ahead and renamed the last css file to sass. So there are no plain css files in the project. I can add the lint for css files back if you think its necessary.

**Pull request merged**